packages/ui/src/components/molecules/Dialog/DialogBox.tsx

```diff
@@ -21,6 +21,31 @@ export interface DialogBoxProps
   children?: ReactNode;
 }
 
+export interface DialogBoxProps extends DialogBoxUtilityProps {
+  $isModal?: boolean;
+}
+
+const modalCss = css`
+  z-index: 100;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+// CSS for modal overlay background
+const ModalOverlay = styled.div`
+  position: fixed;
+  z-index: 99;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  background-color: rgba(0, 0, 0, 0.3);
+  backdrop-filter: blur(2px);
+`;
+
```

Comment:
Can we consider using a dialog element here?

Reply:
Will consider it when building global dialog handler